metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/util/HiveTableUtil.java

```diff
@@ -158,27 +159,15 @@ public static TableIdentifier qualifiedNameToTableIdentifier(final QualifiedName
         return TableIdentifier.parse(name.toString().replace('/', '.'));
     }
 
-    /**
-     * check if the table is a common view.
+    /** check if the table is a common view.
      *
      * @param tableInfo table info
      * @return true for common view
      */
     public static boolean isCommonView(final TableInfo tableInfo) {
-        return tableInfo.getMetadata() != null && isCommonView(tableInfo.getMetadata());
-    }
-
-    /**
-     * check if the table is a common view.
-     *
-     * @param tableMetadata table metadata map
-     * @return true for common view
-     */
-    public static boolean isCommonView(final Map<String, String> tableMetadata) {
-        return tableMetadata != null && Boolean.parseBoolean(tableMetadata.get(DirectSqlTable.COMMON_VIEW));
+        return tableInfo.getMetadata() != null && MetacatUtils.isCommonView(tableInfo.getMetadata());
```

Comment:
It looks to me that this should be 
`tableInfo != null && && MetacatUtils.isCommonView(tableInfo.getMetadata())`;  since the parameter is null will be handled inside the isCommonView function.